austation/code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -5,3 +5,7 @@
 	icon_state = "space_bitters"
 	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
 	foodtype = GRAIN | ALCOHOL
+
+/obj/item/reagent_containers/food/drinks/britcup
+	name = "doctors mug"
+	desc = "A cup with the british flag emblazoned on it. It has the words. Galaxy's best doctor 2530 engrained along the bottom"
```

Comment:
a mug is a type of cup so the description isn't wrong and you cant deny my PR because of that